pkg/management/client.go

```diff
@@ -49,8 +49,8 @@ var (
 	// readinessCheckRetry is used to wait until the API server is reachable
 	readinessCheckRetry = wait.Backoff{
 		Steps:    5,
-		Duration: 10 * time.Millisecond,
-		Factor:   5.0,
+		Duration: 1 * time.Second,
+		Factor:   3.0,
```

Comment:
So, in the last step we'd wait for 3^(5-1) = 81 seconds, right?
Step 1 - 1 sec
...
Step 4 - 27 sec
Step 5 - 81 sec

Reply:
All in all, accumulates to about 6 minutes. Seems reasonable.

Reply:
In the previous implementation, the waiting time would accumulate to about 78 seconds.
Not that bad, no?

Reply:
I think the actual times are a lot shorter. My calculation is the following:

1st retry: 1 second (base duration)
2nd retry: 3 seconds (1 * 3)
3rd retry: 9 seconds (3 * 3)
4th retry: 27 seconds (9 * 3)
5th retry: 81 seconds (27 * 3)

Summing all together, it gives `1 + 3 + 9 + 27 + 81 = 121` seconds total wait time, excluding the effect of jitter. 

Previously it was around `10 ms + 50 ms + 250 ms + 1,250 ms + 6,250 ms = 7,810 milliseconds` (or ~ 7.81 seconds).